### PR TITLE
draft

### PR DESCRIPTION
**Pull request by stefonarch**
Some ideas for updating the about page. In some way the opposite idea of https://github.com/lxqt/lxqt.github.io/pull/56/

schermata-11-15-19-05

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -7,13 +7,38 @@
...
+<li>Lximage-qt - Image Viewer</li>
+<li>QTerminal - Terminal Emulator</li>
+<li>Qps - Process Viewer</li>
+<li>Screengrab - Screen captures</li>
+<li>LXQt-archiver - Archiv Manager</li>
+</ul>
+<br/>
+
+<p>LXQt is available all maior Linux and BSD distributions so you may just try it
+out on your regular system or in a VM.<br>
```
Comment (tsujan) [inline about.html:35]: I think you could drop "regular": "... on your system or in a VM."

File: about.html [hunk 2]
```diff
@@ -7,13 +7,38 @@
...
+More information about installing can be found in the
+<a href="https://github.com/lxqt/lxqt/wiki/">LXQt GitHub Wiki</a>.
 </p>
 <p>
-Historically, LXQt is the product of the merge between LXDE-Qt, an initial Qt flavour of LXDE, and Razor-qt, a project aiming to develop a Qt based desktop environment with similar objectives as the current LXQt.<br>
-LXQt was first supposed to become the successor of LXDE one day but both desktop environments will keep coexisting for the time being.<br>
-More on this topic can be found on page <a href="https://github.com/lxqt/lxqt/wiki/History">History</a> of the aforementioned wiki.
+Historically, LXQt is the product of the merge between LXDE-Qt, an initial Qt
```
Comment (tsujan) [inline about.html:40]: Maybe: "Historically, LXQt **was** the product of the merge..."

**Comment by tsujan** [comment]
The rest is very good, IMO.

**Comment by stefonarch** [comment]
Done, thanks. Nothing to add in the main features?

**Comment by tsujan** [comment]
> Nothing to add in the main features?

If you want to be more specific (which isn't necessary here), you could add:

```
Powerful file manager (that optionally draws the desktop too).
```

You could also add lxqt-panel and lxt-runner to the top of apps list.

**Comment by tsujan** [comment]
And if you want to be still more specific (I don't think you want that):

```
Most LXQt apps work fine under Wayland but LXQt itself does not have a Wayland session yet.
```

**Comment by stefonarch** [comment]
Added the runner, but I think it has to keep a  good balance between short + concise and content.

**Comment by tsujan** [comment]
> I think it has to keep a good balance between short + concise and content.

OK.

File: about.html [hunk 3]
```diff
@@ -7,13 +7,39 @@
...
+
+<h3>Applications</h3>
+
+<ul>
+<li>PcManFm-qt - File Manager</li>
+<li>Lximage-qt - Image Viewer</li>
+<li>QTerminal - Terminal Emulator</li>
+<li>Qps - Process Viewer</li>
+<li>Screengrab - Screen captures</li>
+<li>LXQt-archiver - Archiv Manager</li>
```
Comment (Chiitoo) [inline about.html:30]: 'Archiv' may be missing an 'e' here?

File: about.html [hunk 4]
```diff
@@ -7,13 +7,39 @@
...
+<li>LXQt-runner - Application launcher and calculator</li>
+</ul>
+<br/>
+
+<p>LXQt is available for all maior Linux and BSD distributions so you may just try it
```
Comment (Chiitoo) [inline about.html:35]: Probably 's/maior/major/' here.

Comment (tsujan) [reply about.html:35]: Two pairs of eyes didn't see these!

Comment (tsujan) [reply about.html:35]: I never trust mine and use FeatherPad's spellchecker.

File: about.html [hunk 5]
```diff
@@ -7,13 +7,39 @@
...
+<br/>
+
+<h3>Applications</h3>
+
+<ul>
+<li>PcManFm-qt - File Manager</li>
+<li>Lximage-qt - Image Viewer</li>
+<li>QTerminal - Terminal Emulator</li>
+<li>Qps - Process Viewer</li>
+<li>Screengrab - Screen captures</li>
```
Comment (tsujan) [inline about.html:29]: captures → Capturer; found only by chance.

Comment (Chiitoo) [reply about.html:29]: I almost mentioned that too.  While not exactly wrong, it does "feel" slightly weird compared to the others.

File: about.html (hunk 4 above)
Comment (stefonarch) [reply about.html:35]: This one is from the original text....

File: about.html (hunk 3 above)
Comment (stefonarch) [reply about.html:30]: Too many languages in my poor head, "archiv" in german is correct ;)
Thanks!

File: about.html (hunk 4 above)
Comment (tsujan) [reply about.html]: It's known that the brain automatically corrects misspellings. Good when reading, bad when proofreading.

**Comment by stefonarch** [comment]
Changed one of the two "look and feel" to "appearance".

**Comment by tsujan** [comment]
IMHO, if you don't want to make another change, merge it. If there's a problem, we'll see it.

**Pull request merged**